src/main/kotlin/com/exactpro/th2/common/utils/message/transport/MessageUtils.kt

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2023 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.exactpro.th2.common.utils.message.transport
+
+import com.exactpro.th2.common.event.bean.TreeTable
+import com.exactpro.th2.common.event.bean.TreeTableEntry
+import com.exactpro.th2.common.event.bean.builder.CollectionBuilder
+import com.exactpro.th2.common.event.bean.builder.RowBuilder
+import com.exactpro.th2.common.event.bean.builder.TreeTableBuilder
+import com.exactpro.th2.common.grpc.Direction.FIRST
+import com.exactpro.th2.common.grpc.Direction.SECOND
+import com.exactpro.th2.common.grpc.MessageID
+import com.exactpro.th2.common.message.addField
+import com.exactpro.th2.common.message.toTimestamp
+import com.exactpro.th2.common.schema.message.impl.rabbitmq.transport.Direction.INCOMING
+import com.exactpro.th2.common.schema.message.impl.rabbitmq.transport.GroupBatch
+import com.exactpro.th2.common.schema.message.impl.rabbitmq.transport.Message
+import com.exactpro.th2.common.schema.message.impl.rabbitmq.transport.MessageGroup
+import com.exactpro.th2.common.schema.message.impl.rabbitmq.transport.MessageId
+import com.exactpro.th2.common.schema.message.impl.rabbitmq.transport.ParsedMessage
+import com.exactpro.th2.common.utils.event.transport.toProto
+import com.exactpro.th2.common.utils.message.MessageTableColumn
+import com.exactpro.th2.common.value.toValue
+import java.math.BigDecimal
+import com.exactpro.th2.common.grpc.Message as ProtoMessage
+
+fun Message<*>.toGroup(): MessageGroup = MessageGroup.builder().apply {
+    addMessage(this@toGroup)
+}.build()
+
+fun MessageId.toProto(book: String, sessionGroup: String): MessageID = MessageID.newBuilder().also {
+    it.bookName = book
+    it.direction = if (direction == INCOMING) FIRST else SECOND
```

Comment:
I believe we should use the `when` statement instead (this will help to prevent accidental appears of a new enum value)
```suggestion
    it.direction = when (direction) {
      INCOMING -> FIRST
      OUTGOING -> SECOND
    }
```